discord/guild.py

```diff
@@ -895,6 +896,64 @@ def get_member(self, user_id: int, /) -> Member | None:
         """
         return self._members.get(user_id)
 
+    _FETCHABLE = TypeVar(
```

Comment:
Usually type variables are defined at the very top of a Python file, is there any specific reason it's defined in the Guild class here?